### DISCUSSION TITLE
Deploying Dagster on Azure

### DISCUSSION THREAD

**Discussion by Nielsjonathan** [dagster-io/dagster#18753] [Q&A]
Hey all, I was trying to deploy Dagster on Azure using Azure Container Apps. I tried deploying a container app from the Dagster K8s image on Dockerhub (https://hub.docker.com/r/dagster/dagster-k8s) but with no success. I've found little to no documentation online on how to deploy to Azure so I am hoping that someone from the community has experience with this and is able to give me some pointers on how to get Dagster working on Azure.

**Comment by hwspuijbroek** [comment]
@Nielsjonathan did you receive an answer to this? I am interested in this also but then for the hybrid situation. To use a Dagster Cloud Agent on a Azure Container Apps. I managed to get Dagster Core working with Azure Container apps.
